.github/workflows/main.yml

```diff
@@ -73,6 +73,9 @@ jobs:
       - name: Npm tool install
         run: npm install -g npm@8
 
+      - name: Npm version
+        run: npm version ${{ env.NPM_VERSION }} --no-git-tag-version
```

Comment:
add --allow-same-version
(double check that flag)

Reply:
added --allow-same-version